common/models/job-monitor.js

```diff
@@ -6,72 +6,22 @@ const _ = require('lodash'),
 
 module.exports = function(JobMonitor) {
 
```

Comment:
we no longer clear records downstream from monitors, so remove code related to deleting records.